### DISCUSSION TITLE
自定义组件是否可以覆盖已经部署的工作负载

### DISCUSSION THREAD

**Discussion by wangzd1** [kubevela/kubevela#5045] [Q&A] [answered]
原有的集群中已经部署了一个deployment，是否可以通过vela的application的自定义组件全量覆盖已部署的deployment，而不是通过patch原deployment的逻辑。

**Comment by wonderflow** [comment, marked as answer]
You can specify the feature-gates parameter when your controller start up.

```
--feature-gates=ApplyResourceByUpdate=true
```

This feature gate will replace patch by using update.
